scripts/client_test.sh

```diff
@@ -6,21 +6,25 @@ if [[ "$JOB_TYPE" == "train" ]]; then
     elasticdl train \
       --image_base=elasticdl:ci \
       --model_zoo=model_zoo \
-      --model_def=mnist_functional_api.mnist_functional_api.custom_model \
-      --training_data=/data/mnist/train \
-      --validation_data=/data/mnist/test \
-      --num_epochs=2 \
+      --model_def=deepfm_functional_api.deepfm_functional_api.custom_model \
+      --training_data=/data/frappe/train \
```

Comment:
Any reason to change the dataset for testing?

Reply:
We want to test deepFM model with elasticdl embedding layer using new PS in CI. The frappe dataset is used for deepFM model.